packs/data/equipment.db/staff-of-the-magi.json

```diff
@@ -86,10 +86,9 @@
         "rules": [
             {
                 "key": "FlatModifier",
-                "label": "Staff of the Magi (vs. spells)",
                 "predicate": {
-                    "all": [
-                        "spells"
+                    "any": [
```

Comment:
Why change to any? There is only one option. It is still effectively an all...

Reply:
Oh, just copy paste. Nothing specific.

Reply:
```suggestion
                    "all": [
```